src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.enums.Color;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double top;
+    private double bottom;
+    private double height;
+
+    public IsoscelesTrapezoid(Color color, double top, double bottom, double side) {
+        super(color);
+        this.top = top;
+        this.bottom = bottom;
+        this.height = side;
```

Comment:
The constructor parameter `side` is misleading as it is used to set the `height`. Consider renaming it to `height` for clarity.